app/uk/gov/hmrc/incometaxsubscriptionselfemployedfrontend/config/AddressLookupConfig.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxsubscriptionselfemployedfrontend.config
+
+import javax.inject.Inject
+import play.api.i18n.{Lang, MessagesApi}
+
+class AddressLookupConfig @Inject()(messagesApi: MessagesApi) {
+
+  //scalastyle:off
+  def config(continueUrl: String)(implicit language: Lang): String = {
+
+    val cy = Lang("CY")
+
+    s"""{
+       |  "version": 2,
+       |  "options": {
+       |    "continueUrl": "$continueUrl",
+       |    "showBackButtons": true,
+       |    "includeHMRCBranding": true,
+       |    "ukMode": true,
+       |    "selectPageConfig": {
+       |      "proposalListLimit": 50,
+       |      "showSearchLinkAgain": true
+       |    },
+       |    "confirmPageConfig": {
+       |      "showChangeLink": true,
+       |      "showSubHeadingAndInfo": true,
+       |      "showSearchAgainLink": false,
+       |      "showConfirmChangeText": true
+       |    },
+       |    "timeoutConfig": {
+       |      "timeoutAmount": 890,
```

Comment:
can you please change this to 900s instead of 890 , it is not s big issue, but as all our service is 900 for the consistency and accessibility purpose. I also found in the https://github.com/hmrc/address-lookup-frontend they use 900 s too.

Reply:
Done